src/components/Card/Card.stories.tsx

```diff
@@ -372,3 +373,37 @@ export const LastItemAlignment = () => (
     </Card>
   </Grid>
 );
+
+const CustomLink = styled.a`
+  border: 2px solid purple !important;
+`;
+
+export const InteractiveButtonCard = () => (
+  <Stack gap="lg">
+    <Heading size="h2">Interactive button card</Heading>
```

Comment:
Is there any specific reason to have the headings inside stories? If not I would remove those, story name should be enough.